src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -309,6 +310,12 @@ void SQLPipelineStatement::_precheck_ddl_operators(const std::shared_ptr<Abstrac
       AssertInput(file.good(), "There is no file '" + import->filename + "'.");
       break;
     }
+    /*case OperatorType::Export: {
+      const auto export_table = std::dynamic_pointer_cast<Export>(pqp);
+      AssertInput(storage_manager.has_table(export_table->tablename),
+                  "There is no table '" + export_table->tablename + "'.");
+      break;
+    }*/
```

Comment:
Is this till WIP?

Reply:
Exactly, this is related to my question above. Giving the operator another operator as input vs. a table name decides on where we check if this table exists. With a table name we can check here, with an operator we need either to check in LQPTranslator or even SQLTranslator.